_posts/2013-04-11-testing.md

```diff
@@ -324,3 +324,30 @@ Once your helpers are defined, you'll want to ensure that they are listed in the
     "myGreatHelper": true,
     "myGreatAsyncHelper": true
 {% endhighlight %}
+
+#### Test Options
+
+The Ember Test command has options that you can use to change the default behavior the command. Options can be prefixed with `--no-` to invert their value. For example, the command
+
+`ember test -s --no-launch`
+
+will launch a testing server without opening a new browser window.
+
+|Name|Type|Default|Aliases|Description|
+|---|---|---|---|---|
+|environment|String|test|e|Possible values are "development", "production", and "test".|
+|config-file|String||c,cf|
+|server|Boolean|false|s|
+|host|String|localhost|H|
+|test-port|Number|7357|tp|The test port to use when running tests. Pass 0 to automatically pick an available port|
+|filter|String||f|A string to filter tests to run|
+|module|String||m|The name of a test module to run|
+|watcher|String|events|w|
+|launch|String|false||A comma separated list of browsers to launch for tests.|
+|reporter|String|r||Test reporter to use tap|
```

Comment:
this is the output for `reporter` when running `ember -h`:

```
--reporter (String) Test reporter to use [tap|dot|xunit] (default: tap)
```

so perhaps the default here should be `tap` rather than `r`.